index.html

```diff
@@ -24,99 +24,110 @@
+        <span>Технік з автоматизації</span>
 
-        <div class="contact_icon">
-            <a href="https://www.linkedin.com/in/taras-mavrovich-6ba943241/" target="_blank" class="reviews_star">
-                <i class="fa-brands fa-linkedin-in" aria-hidden="true"></i>
-            </a>
+      </div>
 
-            <a href="https://www.facebook.com/profile.php?id=100017592820743" target="_blank" class="reviews_star">
-                <i class="fa-brands fa-facebook-f" aria-hidden="true"></i>
-            </a>
+      <!-- my_sidebar_cells -->
+      <div class="my_sidebar_cells">
+        <!-- contact -->
+        <div class="contact contacts">
 
-            <a href="https://t.me/taracccc" target="_blank" class="reviews_star">
-                <i class="fa-brands fa-telegram" aria-hidden="true"></i>
-            </a>
+          <h2 class="contact_contacts helpers_titles">Контакти</h2>
 
-            <a href="viber://chat?number=+380664516911" target="_blank" class="reviews_star">
-                <i class="fa-brands fa-viber" aria-hidden="true"></i>
-            </a>
-        </div>
+          <div class="contact_icon">
+              <a href="https://www.linkedin.com/in/taras-mavrovich-6ba943241/" target="_blank" class="reviews_star">
+                  <i class="fa-brands fa-linkedin-in" aria-hidden="true"></i>
+              </a>
 
-        <div>
-            <a href="tel:+380664516911" class="contact_tel">+380664516911</a>
-        </div>
+              <a href="https://www.facebook.com/profile.php?id=100017592820743" target="_blank" class="reviews_star">
+                  <i class="fa-brands fa-facebook-f" aria-hidden="true"></i>
+              </a>
+
+              <a href="https://t.me/taracccc" target="_blank" class="reviews_star">
+                  <i class="fa-brands fa-telegram" aria-hidden="true"></i>
+              </a>
+
+              <a href="viber://chat?number=+380664516911" target="_blank" class="reviews_star">
+                  <i class="fa-brands fa-viber" aria-hidden="true"></i>
+              </a>
+          </div>
+
+          <div>
+              <a href="tel:+380664516911" class="contact_tel">+380664516911</a>
+          </div>
+
+          <div>
+              <a href="mailto:taras.mavrovich1998@gmail.com" class="contact_email">taras.mavrovich1998@gmail.com</a>
+          </div>
 
-        <div>
-            <a href="mailto:taras.mavrovich1998@gmail.com" class="contact_email">taras.mavrovich1998@gmail.com</a>
         </div>
+        <!-- /contact -->
+
+        <!-- skills -->
+        <div class="my_sidebar_sections skills">
+          <h2 class="helpers_titles">Навички</h2>
+
+          <h3 class="helpers_sub_titles">Професійні</h3>
+          <ul class="some_list">
+            <li class="tag_list">Дослідження</li>
+            <li class="tag_list">Аналітика</li>
+            <li class="tag_list">Розвиток виробництва</li>
+            <li class="tag_list">Якісність продукції</li>
```

Comment:
A lot of repeats: class="tag_list"
You can remove this class and make it easier